src/status_im/multiaccounts/core_test.cljs

```diff
@@ -0,0 +1,85 @@
+(ns status-im.multiaccounts.core-test
+  (:require [cljs.test :refer-macros [deftest is testing]]
+            [status-im.multiaccounts.core :as ma]))
+
+(def pk "0x0461f576da67dc0bca9888cdb4cb28c80285b756b324109da94a081585ed6f007cf00afede6b3ee5638593674fee100b590318fc7bdb0054b8dd9445acea216ad2")
+(def short-pk "0x0461…ad2")
+(def random-name "Studious Gold Mustang")
+(def override-random-name (str "override" random-name))
+(def nickname "nickname")
+(def override-nickname (str nickname "override"))
+(def display-name "display-name")
+(def override-display-name (str display-name "override"))
+(def ens-name "jakubgs.eth")
+(def formatted-ens (str "@" ens-name))
+(def override-ens-name (str "override" ens-name))
+
+(def contact
+  {:nickname     nickname
+   :name         ens-name
+   :ens-verified true
+   :public-key   pk
+   :display-name display-name})
+
+(deftest contact-two-names
```

Comment:
FYI: It's more correct to add the `-test` suffix to tests because the macro `deftest` will create a var with the name `contact-two-names` and add the macro body as metadata. Let's say, in more _busy_ namespaces, not adding the suffix can lead to `deftest` calls replacing functions previously defined in the same namespace, so that's why the general recommendation is to add the suffix. I wish the linter could catch this by default.